Makefile

```diff
@@ -0,0 +1,177 @@
+# Makefile for Sphinx documentation
```

Comment:
Sorry, I will describe how-to-build in `README.md`. I prepared `Dockerfile` so we can build document without installing modules in local.